plugins/obs-outputs/mp4-mux.h

```diff
@@ -22,6 +22,13 @@
 
 struct mp4_mux;
 
+/* Flavour for target compatibility */
+enum mp4_flavour {
+	FLAVOUR_MP4,  /* ISO/IEC 14496-12 */
+	FLAVOUR_MOV,  /* Apple QuickTime */
+	FLAVOUR_CMAF, /* ISO/IEC 23000-19 (not yet implemented) */
+};
+
```

Comment:
If moving from an internal header to a non-internal header, perhaps we should change `flavour` to `flavor` to match the rest of the codebase.